tasty-query/shared/src/main/scala/tastyquery/TypeTrees.scala

```diff
@@ -121,7 +121,7 @@ object TypeTrees {
       extends TypeTree(span)
       with DefTree(symbol) {
     override protected def calculateType(using Context): Type =
-      TermRef(NoType, symbol)
+      TypeRef(NoType, symbol)
```

Comment:
TypeTreeBind is for a match type right? so this change doesn't seem to make sense